software/local-content-share.yml

```
@@ -0,0 +1,12 @@
+name: Local Content Share
+website_url: https://github.com/Tanq16/local-content-share
+description:  Browser frontend that enables share and storing text snippets and files wit any local network devices. 
```

```suggestion
description:  Browser frontend that enables share and storing text snippets and files with any local network devices. 
```
Minor grammatical correction